scripts/simulate-density-model.js

```
@@ -0,0 +1,567 @@
+  const widths = headers.map((h, i) =>
+    Math.max(
+      h.length,
+      ...body.map((row) => String(row[i]).length)
+    )
+  );
+
+  const fmt = (row) => row.map((v, i) => String(v).padEnd(widths[i])).join(' | ');
+
+  console.log('\n' + fmt(headers));
+  console.log(widths.map((w) => '-'.repeat(w)).join('-|-'));
+  body.forEach((row) => console.log(fmt(row)));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)